docs/dive-in/2024/01/2024-01-16:update-make-file.md

```diff
@@ -0,0 +1,14 @@
+# Make
+
+Make is a utility to automate somme commands.  
+Also it allows to have a kind of consistency between commands or to simplify commands.  
+It allows with a single command to run that will run multiple commands.
+
+Those commands are registered in the `Makefile` and have the same skeleton:  
+`make [commandName] [arguments]`
```

Comment:
What kind of arguments will you put ?

Reply:
For the commands of the tasks there won't be any arguments